Description: Fix FTBFS in cwidget as well as in aptitude due to missing #include <pthread.h>
Author: Paul Wise <pabs@debian.org>
Reviewed-By: Axel Beckert <abe@debian.org>
Bug-Debian: https://bugs.debian.org/1015925
Bug: https://bugs.debian.org/1015925

--- a/src/cwidget/generic/threads/threads.h
+++ b/src/cwidget/generic/threads/threads.h
@@ -26,6 +26,7 @@
 #define THREADS_H
 
 #include <errno.h>
+#include <pthread.h>
 #include <cwidget/generic/util/exception.h>
 
 namespace cwidget
